resources/qml/dialogs/ReadReceipts.qml

```diff
@@ -54,23 +54,29 @@ ApplicationWindow {
                 id: readReceiptsList
 
                 clip: true
-                spacing: Nheko.paddingMedium
                 boundsBehavior: Flickable.StopAtBounds
                 model: readReceipts
 
                 delegate: ItemDelegate {
+                    id: del
+
                     onClicked: room.openUserProfile(model.mxid)
                     padding: Nheko.paddingMedium
-                    width: receiptLayout.implicitWidth
-                    height: receiptLayout.implicitHeight
+                    width: ListView.view.width
+                    height: receiptLayout.implicitHeight + Nheko.paddingSmall * 2
                     hoverEnabled: true
                     ToolTip.visible: hovered
                     ToolTip.text: model.mxid
+                    background: Rectangle {
+                        color: del.hovered ? Nheko.colors.dark : readReceiptsRoot.color
+                    }
 
                     RowLayout {
                         id: receiptLayout
 
                         spacing: Nheko.paddingMedium
+                        anchors.verticalCenter: parent.verticalCenter
+                        x: parent.x + Nheko.paddingSmall
```

Comment:
anchors.centerIn: parent maybe?

Reply:
Nope, that centers it in the window and I can confirm that that looks weird.

Reply:
What. But you center in the parent vertically already!

Reply:
But I don't want to center horizontally.

Reply:
What do you want to do then? Add padding?

Reply:
Yes

Reply:
Can't you use the padding property then? Otherwise you are just shifting the whole item to the right, which might cause it to go out of bounds.